services/Cargo.toml

```diff
@@ -12,7 +12,7 @@ edition = "2018"
 chrono = { version = "0.4", features = ["serde"] }
 geoengine-datatypes = { path = "../datatypes" }
 geoengine-operators = { path = "../operators" }
-tokio = { version = "0.2", features = ["macros"] }
+tokio = { version = "0.2", features = ["macros", "signal"] }
```

Comment:
dev-dependency only or add add it to main.rs as well. might make sense

Reply:
928702e